integration-tests/run

```diff
@@ -67,10 +67,14 @@ integration-test/common:
 	git fetch --depth=1 https://github.com/cockpit-project/cockpit.git 191
 	git archive FETCH_HEAD -- test/common | tar -x -C integration-tests --strip-components=1 -f -
 
-node_modules/%:
+node_modules:
+	mkdir node_modules
+
+node_modules/%: node_modules
 	npm install $*
 
 cockpit/node_modules:
+	mkdir cockpit/node_modules
```

Comment:
why is the explicit mkdir necessary?

Reply:
It is not strictly necessary.  I think npm actually searches upwards for either node_modules or package.json, and since cockpit/ contains package.json it will itself create node_modules there.
I just thought that this is more explicit.